tests/test_jurassic_tokenizer.py

```diff
@@ -25,6 +25,10 @@ def test_tokenizer_encode_set(tokenizer: JurassicTokenizer, resources_path: Path
                 tokenized_doc["doc_text"]
             ), f"Not equal at doc {i}"
 
+            assert tokenized_doc["token_ids_start_false"] == tokenizer.encode(
```

Comment:
Can you please move this to a separated test called something like `test_tokenizer_encode_set__when_is_start_<true/false>__should_<do_what_what_you_expect>`? 

This would allow us to pin point the issue/changes if anything changes

Reply:
Done. Couldn't really add the `should_<do_what_what_you_expect>` part since we basically assert the results are what is saved in the `resources_path`